webui/src/custom-mui-components/error-dialog.tsx

```diff
@@ -13,12 +13,26 @@ import { Dialog, DialogTitle, DialogContent, Button, DialogContentText, DialogAc
 
 export class ErrorDialog extends React.Component<ErrorDialog.Props, {}> {
 
+    handleEnter = (e: KeyboardEvent) => {
+        if (e.keyCode === 13) {
```

Comment:
`keyCode` is deprecated, please use `code` and check for values `'Enter'` and `'Escape'`.

https://www.w3.org/TR/uievents-code/#key-alphanumeric-functional

Reply:
Thanks! I have swaped keyCode with code.